effects/CMakeLists.txt

```diff
@@ -18,16 +18,10 @@ else (APPLE)
       set(INCS "")
 endif (APPLE)
 
-if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
-      find_package(Qt5QuickCompiler)
+find_package(Qt5QuickCompiler)
       qtquick_compiler_add_resources (qrc_effects_files
             zita1/zita.qrc
             )
```

Comment:
Shouldn't this be unindented?

Reply:
No. It wasn't indented far enough before IMHO

Reply:
Actually yes, sorry